apps/theme/components/Sidebar/ColorPane/ColorPane.tsx

```diff
@@ -53,19 +57,26 @@ export const ColorPane = ({
         >
           <ChevronLeftIcon title='a11y-title' fontSize='1.5rem' /> Gå tilbake
         </Button>
-        <Button
-          data-size='sm'
-          variant='tertiary'
-          data-color='danger'
-          onClick={() => onRemove()}
-          className={cl(
-            classes.removeBtn,
-            (type !== 'editColor' || colorType === 'neutral') && classes.hide,
-          )}
+        <Tooltip
+          content='Du må ha minst en hovedfarge'
+          hidden={colorType !== 'main' || mainColors.length > 1}
         >
-          Fjern farge
-          <TrashIcon title='a11y-title' fontSize='1.5rem' />
-        </Button>
+          <Button
+            data-size='sm'
+            variant='tertiary'
+            data-color='danger'
+            onClick={() => {
+              if (colorType === 'main' && mainColors.length === 1) return;
```

Comment:
Suggest moving this out to a const with descriptive name, something like; `hasOneMainColor`